gamebook/game_manager.py

```diff
@@ -0,0 +1,43 @@
+from gamebook.terminal_output import TerminalOutput
+from gamebook.terminal_input import TerminalInput
+import sys
+
+
+class GameManager(object):
+    def __init__(self, scene):
+        self.scene = scene
+
+    def start(self, scene_import):
+        current_scene = self.scene
+        while True:
+            next_scene_name = current_scene.run_scene()
+            current_scene = self.next_scene(next_scene_name, scene_import)
+
+    def next_scene(self, scene_name, scene_import):
+        for scene in scene_import:
+            if scene.get_name() == scene_name:
+                return scene
+
+    def run_scene(self):
+        output_instance = TerminalOutput()
+        input_instance = TerminalInput()
+
+        output_instance.output(self.scene.show_desc())
+
+        if len(self.scene.options) == 0:
+            sys.exit()
+
+        user_input = input_instance.input("your choice: ")
```

Comment:
Isn't this something that the `Scene` takes care of? do you want it to write "your choice" every single time?

Reply:
scene doesn't take care of that. i do want it to write something like "your choice:" so they will see a place to insert the user_input